mono/metadata/reflection.c

```diff
@@ -1519,6 +1521,7 @@ assembly_name_to_aname (MonoAssemblyName *assembly, char *p) {
 		*p = 0;
 		p++;
 	}
+	g_strchomp (assembly->name);
```

Comment:
Over in the https://github.com/dotnet/runtime/pull/37156 (mirror PR for the dotnet/runtime version of Mono), the builds are failing here.
```
/__w/1/s/src/mono/mono/metadata/reflection.c:1524:14: error: passing 'const char *' to parameter of type 'gchar *' (aka 'char *') discards qualifiers [-Werror,-Wincompatible-pointer-types-discards-qualifiers]
          g_strchomp (assembly->name);
                      ^~~~~~~~~~~~~~
  /__w/1/s/src/mono/mono/eglib/glib.h:377:39: note: passing argument to parameter 'str' here
  gchar       *g_strchomp       (gchar *str);
                                        ^
  1 error generated.
```
```suggestion
	g_strchomp ((char*)assembly->name);
```